src/sign.ts

```diff
@@ -89,6 +90,15 @@ async function setupPlugin() {
     }
 }
 
+function getPluginConfigList(pluginConfig: string): string[] {
+    let pluginConfigList: string[] = [];
+    for (const config of pluginConfig.split(/\r|\n/)) {
```

Comment:
What will happen to `pluginConfig.split(/\r|\n/)` given `foo=1\r\nbar=2` or `foo=1\r\n\r\nbar=2\r\n`?

Reply:
Updated to JSON array string as input so the logic no longer needs to parse \r or \n.